src/Games/NexusMods.Games.FOMOD/CoreDelegates/UiDelegate.cs

```diff
@@ -1,11 +1,11 @@
 using FomodInstaller.Interface;
-using FomodInstaller.Interface.ui;
 using Microsoft.Extensions.Logging;
-using NexusMods.Common.UserInput;
+using NexusMods.Common.GuidedInstaller;
+using NexusMods.Common.GuidedInstaller.ValueObjects;
 
 namespace NexusMods.Games.FOMOD.CoreDelegates;
 
-public class UiDelegates : IUIDelegates
+public class UiDelegates : FomodInstaller.Interface.ui.IUIDelegates
```

Comment:
Any way to avoid the full qualifier each time?

Reply:
No, I made those explicit in this file to show what's from the external library and what's ours.